packages/block-editor/src/components/block-settings/container.native.js

```diff
@@ -3,13 +3,13 @@
  */
 import {
 	InspectorControls,
+	ImageLinkDestinationsScreen,
 	useMultipleOriginColorsAndGradients,
 } from '@wordpress/block-editor';
```

Comment:
I realize some of these imports were already in place, but my perception is that the project generally prefers importing internal dependencies. We could technically change these to local imports.